mobile/src/main/java/ch/epfl/sdp/mobile/ui/profile/UserScreen.kt

```diff
@@ -32,7 +32,7 @@ fun UserScreen(
     profileTabBar: @Composable () -> Unit,
     matches: List<ChessMatch>,
     lazyColumnState: LazyListState,
-    modifier: Modifier = Modifier,
+    modifier: Modifier,

```

Comment:
Question : why did you remove this default `Modifier` ?

Reply:
Not needed and to improve test coverage